jerry-ext/debugger/debugger-tcp.c

```diff
@@ -252,7 +254,7 @@ jerryx_debugger_tcp_configure_socket (jerryx_socket server_socket, /** < socket
 
   int opt_value = 1;
 
-  if (setsockopt (server_socket, SOL_SOCKET, SO_REUSEADDR, &opt_value, sizeof (int)) != 0)
+  if (setsockopt (server_socket, SOL_SOCKET, SO_REUSEADDR, (const char *) &opt_value, sizeof (int)) != 0)
```

Comment:
The signature of `setsockopt` is also platform-dependent.
Windows: `int setsockopt(SOCKET s, int level, int optname, const char *optval, int optlen);`
Linux: `int setsockopt(int sockfd, int level, int optname, const void *optval, socklen_t optlen);`

So, I think it would be good to add yet another typedef, `jerryx_socket_opt_t` (extend the code suggested in the previous comment).